projects/ontimize-web-ngx/src/lib/components/table/extensions/contextmenu/o-table-context-menu.component.ts

```diff
@@ -361,4 +369,8 @@ export class OTableContextMenuComponent implements AfterViewInit {
     this.table.dataSource.setRowGroupLevelExpansion(this._row, false);
 
   }
+  isColumnDate() {
+
+    this.isDateColumn.next(this.column.sqlType == 93);
```

Comment:
This way is not fully correct. The types 91 or 92 are also relative to dates. We have to found another way of identifying date columns